src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -186,7 +186,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <StaticICULibs Include="-Wl,-Bdynamic" Condition="'$(StaticExecutable)' != 'true'" />
     </ItemGroup>
 
-    <ItemGroup Condition="'$(StaticOpenSslLinking)' != 'true' and Exists('$(IlcSdkPath)nonportable.txt')">
+    <ItemGroup Condition="'$(StaticOpenSslLinking)' != 'true' and Exists('$(IlcSdkPath)nonportable.txt') and '$(_targetOS)' == 'linux'">
```

Comment:
This should not be limited to Linux. This should be enabled for all Unix platforms that use OpenSSL that's all Unixes except Apple and Android.

I think the actual problem here is that the brew packages are built as non-portable builds. Why are the brew packages built as nonportable builds? Would it make more sense to build the brew packages as portable builds?

Reply:
The non-portable builds are expected to have their own RID and be locked to the specific OS/distro version that it was built on. I do not think any of these properties are desirable for brew distributions.

Reply:
Why would it not be "okay" to use non-portable for brew? They do a build per OS, per arch.

https://github.com/Homebrew/homebrew-core/blob/db0ac99bc2e63ee6f9613eaf23ad8790cb99a6c5/Formula/d/dotnet.rb#L31-L35

So it seems (to me) fine to use non-portable builds.

Reply:
The linux support for brew there though looks suspicious though. They are building non-portable but only really have "a linux".